### PR TITLE
MC-10462 Add API doc for Reset password in Azure

### PR DESCRIPTION
**Pull request by GuillaumeVialle**
Added doc for reset password.

Screen Shot 2020-01-28 at 11 02 30 AM

### CODE REVIEW

File: source/includes/azure/_instances.md [hunk 1]
```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
 ------ | -----------
 `machineType`<br/>*string* | The new machine type to assign to the instance
 
+<!-------------------- RESET PASSWORD -------------------->
+
+#### Reset password
+
+The admin password is used to access Linux or Windows [instances](#azure-instances). 
+
+```shell
```
Comment (fyrnaga) [inline source/includes/azure/_instances.md:259]: It should say :
Reset the administrator password on a Windows or Linux [instances] (#azure-instances).

File: source/includes/azure/_instances.md [hunk 2]
```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
...
+  "username": "johndoe",
+  "password": "SomePassw0rdVal!d"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id?operation=reset_password</code>
+
+Update the administrator username for the existing instance, or insert a new user.
+
+<aside class="warning">
```
Comment (fyrnaga) [inline source/includes/azure/_instances.md]: Not sure if we want to document this

**Review by fyrnaga — CHANGES_REQUESTED** [review]
Small change and suggestion before getting this merge

File: source/includes/azure/_instances.md [hunk 3]
```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
 ------ | -----------
 `machineType`<br/>*string* | The new machine type to assign to the instance
 
+<!-------------------- RESET PASSWORD -------------------->
+
+#### Reset password
+
+The admin password is used to access Linux or Windows [instances](#azure-instances).
```
Comment (evanluc) [inline source/includes/azure/_instances.md]:
```suggestion
Reset the administrator password on a Windows or Linux [instances](#azure-instances). 
```

File: source/includes/azure/_instances.md [hunk 4]
```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
...
+```json
+{
+  "username": "johndoe",
+  "password": "SomePassw0rdVal!d"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id?operation=reset_password</code>
+
+Update the administrator username for the existing instance, or insert a new user.
```
Comment (evanluc) [inline source/includes/azure/_instances.md]:
```suggestion
Update the administrator username for the existing instance, or insert a new user if it doesn't exist on the instance.
```

And then maybe we can remove the warning?

File: source/includes/azure/_instances.md [hunk 5]
```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
...
+
+<aside class="warning">
+If the username is different than the admin username, it will insert a new user.
+</aside>
+
+Required | &nbsp;
+------ | -----------
+`username`<br/>*string* | The administrator username of the instance or a new username. It cannot be a reserve user such as admin,root or administrator and must not be more than 20 characters.
```
Comment (simongodard) [inline source/includes/azure/_instances.md]: > It cannot be a **reserved username** admin, root or administrator...

Comment (simongodard) [reply source/includes/azure/_instances.md]: missing space before *root*

File: source/includes/azure/_instances.md [hunk 6]
```diff
@@ -250,6 +250,42 @@ Required | &nbsp;
...
+`password`<br/>*string* | The password of the account. It must be between between 12 and 72 characters and must be a combination of 3 of the following patterns : Special lettes, Upper Case, Lower Case and Numbers.
```
Comment (simongodard) [inline source/includes/azure/_instances.md]: - What do you mean by `Special lettes`? `Special characters`?
- Upper Case -> Uppercase
- Lower Case -> Lowercase

File: source/includes/azure/_instances.md (hunk 3 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_instances.md]: Changed it

File: source/includes/azure/_instances.md (hunk 1 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_instances.md:259]: Done!

File: source/includes/azure/_instances.md (hunk 6 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_instances.md]: I copied from create instance and just changed it for both!

File: source/includes/azure/_instances.md (hunk 5 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_instances.md]: done in both create instance and reset password.

File: source/includes/azure/_instances.md (hunk 2 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_instances.md]: I removed it and added a bigger explanation.

File: source/includes/azure/_instances.md (hunk 4 above)
Comment (GuillaumeVialle) [reply source/includes/azure/_instances.md]: Just refactored the entire paragraph.

**Review by fyrnaga — APPROVED** [review]
LGTM

**Review by simongodard — APPROVED** [review]

**Pull request merged**